tests/python/rest_api/test_tasks.py

```diff
@@ -1568,7 +1573,8 @@ def test_can_rename_label(self, tasks, labels, admin_user):
         resulting_labels = self._get_task_labels(task["id"], admin_user)
         assert DeepDiff(resulting_labels, task_labels, ignore_order=True) == {}
 
-    def test_cannot_rename_label_to_duplicate_name(self, tasks, labels, admin_user):
+    def test_cannot_rename_label_to_duplicate_name(self, tasks_wlc, labels, admin_user):
+        tasks = tasks_wlc
```

Comment:
IMO, renamings like these serve to obscure the underlying data structure. If a variable is supposed to have label counts, then let it keep the `_wlc` suffix.